### PR TITLE
Feature flex 1666 adding custom attribute

### PR DESCRIPTION
**Pull request by girishdudhe**
Adding custom attributes(data-qa) in common components...

### CODE REVIEW

File: src/accordian/Accordion.tsx [hunk 1]
```diff
@@ -13,11 +13,19 @@ import {classNames} from "../utils";
 import {ClassNames} from "./ClassNames";
 import {Icon, Direction} from "../icon";
 
+/**
+ * Accordian Props
+ * style: css style
+ * className: css property
+ * accordianMultiPanel: if yes multipanel enebled else not
+ * dataqa: quality engineering testing field
```
Comment (akate1) [inline src/accordian/Accordion.tsx]: use below format for documentation
@param {dataqa} quality engineering tag

File: src/emphasis/badges/Badges.tsx [hunk 2]
```diff
@@ -11,9 +11,16 @@
 import * as React from "react";
 import {classNames} from "../../utils";
 
+/**
+ * Badge Props
+ * color: color of label
+ * status: status of label
+ * dataqa: quality engineering testing field
```
Comment (akate1) [inline src/emphasis/badges/Badges.tsx]: use below format for documentation
@param {dataqa} quality engineering tag

File: src/forms/checkbox/CheckBox.tsx [hunk 3]
```diff
@@ -12,6 +12,19 @@ import * as React from "react";
...
+ * label: label of checkbox
+ * name: name of checkbox
+ * id: id of checkbox
+ * checked: if true checked else false
+ * defaultChecked: default checked value
+ * ariaLabel: arial label
+ * ariaLabelledby: arial label details
+ * disabled: property to enable disable checkbox
+ * className: css property
+ * dataqa: quality engineering testing field
```
Comment (akate1) [inline src/forms/checkbox/CheckBox.tsx]: use below format for documentation
@param {dataqa} quality engineering tag

File: src/forms/datepicker/DatePicker.tsx [hunk 4]
```diff
@@ -3,10 +3,18 @@ import {Icon, Direction} from "../../icon";
 import Moment from "moment";
 import {classNames} from "../../utils";
 
+/**
+ * DatePicker Props
+ * value: date value
+ * locale: regional code
+ * defaultValue: default date value
+ * dataqa: quality engineering testing field
```
Comment (akate1) [inline src/forms/datepicker/DatePicker.tsx]: use below format for documentation
@param {dataqa} quality engineering tag

File: src/forms/dropdown/Dropdown.tsx [hunk 5]
```diff
@@ -18,12 +18,22 @@ import {ReactElement, ReactNode} from "react";
...
 import {STOP_PROPAGATION} from ".";
 
+/**
+ * DropDown Props
+ * label: label of checkbox
+ * isNested: property to nest dropdown
+ * className: css property
+ * button: dropdown button
+ * showCaret: caret property
+ * dataqa: quality engineering testing field
```
Comment (akate1) [inline src/forms/dropdown/Dropdown.tsx]: use below format for documentation
@param {dataqa} quality engineering tag

File: src/forms/label/Label.tsx [hunk 6]
```diff
@@ -12,13 +12,23 @@ import * as React from "react";
...
 import {Icon} from "../../icon";
 
+/**
+ * Label Props
+ * color: color of label
+ * status: status of label
+ * dismissable: if true label is dismissable
+ * style: css style
+ * className: css property
+ * dataqa: quality engineering testing field
```
Comment (akate1) [inline src/forms/label/Label.tsx]: use below format for documentation
@param {dataqa} quality engineering tag

File: src/forms/radio/RadioButton.tsx [hunk 7]
```diff
@@ -12,6 +12,20 @@ import * as React from "react";
...
+ * className: css property
+ * disabled: property to enable disable checkbox
+ * label: label of checkbox
+ * labelclass: label css style
+ * name: name of checkbox
+ * value: value of radio button
+ * children: nested radio button or group
+ * style: css style
+ * inButtonGroup: button group property
+ * dataqa: quality engineering testing field
```
Comment (akate1) [inline src/forms/radio/RadioButton.tsx]: use below format for documentation
@param {dataqa} quality engineering tag

File: src/forms/radio/RadioButtonGroup.tsx [hunk 8]
```diff
@@ -13,6 +13,19 @@ import {RadioButton} from "./RadioButton";
...
+ * defaultValue: default value of radio group
+ * children: nested radio button or group
+ * className: css property
+ * disabled: property to enable disable radio button group
+ * checked: if true checked else false
+ * helperText: helper text of radio group
+ * inline: inline style
+ * label: label of radio button group
+ * name: name of radio group
+ * dataqa: quality engineering testing field
```
Comment (akate1) [inline src/forms/radio/RadioButtonGroup.tsx]: use below format for documentation
@param {dataqa} quality engineering tag

File: src/progress_bars/ProgressBars.tsx [hunk 9]
```diff
@@ -11,7 +11,18 @@
...
+ * ProgressBar Props
+ * value: value of progress
+ * max: maximum progress value
+ * labeled: if true label appears
+ * style: css style
+ * className: css property
+ * status: status of label
+ * type: type of progressbar
+ * position: progress Bar position
+ * dataqa: quality engineering testing field
```
Comment (akate1) [inline src/progress_bars/ProgressBars.tsx]: use below format for documentation
@param {dataqa} quality engineering tag

File: src/tables/Table.tsx [hunk 10]
```diff
@@ -12,11 +12,20 @@ import * as React from "react";
 import {classNames} from "../utils";
 import {ClassNames} from "../tables";
 
+/**
+ * Table Props
+ * className: css property
+ * isVertical: table orientation
+ * isCompact: compaction property
+ * isNonBordered: Border property
+ * dataqa: quality engineering testing field
```
Comment (akate1) [inline src/tables/Table.tsx]: use below format for documentation
@param {dataqa} quality engineering tag

File: src/accordian/Accordion.tsx (hunk 1 above)
Comment (girishdudhe) [reply src/accordian/Accordion.tsx]: Addressed

File: src/forms/checkbox/CheckBox.tsx (hunk 3 above)
Comment (girishdudhe) [reply src/forms/checkbox/CheckBox.tsx]: Addressed

File: src/emphasis/badges/Badges.tsx (hunk 2 above)
Comment (girishdudhe) [reply src/emphasis/badges/Badges.tsx]: Addressed

**Pull request merged**